### PR TITLE
Features page header a/b test

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description
5%/5% a/b test for removing the heading from the features page.

## Issue / Bugzilla link
First part of #6155

## Testing
- experiment loads
- no JS errors

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Yeah, the automation assumption is something we're trying to test here 🤞

**Comment by stephaniehobson** [comment]
Updated to skip test. Ready for re-review.

File: tests/functional/test_internet_health.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 @pytest.mark.smoke
 @pytest.mark.nondestructive
+@pytest.mark.skipif(reason='https://github.com/mozilla/bedrock/pull/6139')
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply tests/functional/test_internet_health.py]: 🤦‍♀️

**Comment by stephaniehobson** [comment]
Ready for re-re-re-review.

**Comment by alexgibson** [comment]
I already approved this, so feel free to merge!

**Pull request merged**